app/views/orphans/_orphan_pseud.html.erb

```diff
@@ -1,22 +1,43 @@
 <!--Descriptive page name, messages and instructions-->
-<h2 class="heading"><%= ts('Orphan All Works by') %> <%= pseud.name %></h2>
+<% works = pseud.works.to_a %>
+<h2 class="heading"><%= ts("Orphan All Works by %{name}", name: pseud.name) %></h2>
 
 <p class="caution">
-<%= ts("Are you sure you want to") %>
-<strong><%= ts("permanently") %></strong>
-<%= ts("remove all identifying information from all the works you've posted to 
-the archive under this pseud, along with their chapters, 
-associated series and any feedback replies you may have left?") %>
+  <%= ts("Orphaning all works by %{name} will", name: pseud.name)%>
+  <strong><%= ts("permanently")%></strong>
+  <%= ts("detach the pseud %{name} from the following work(s), their chapters, associated series, and any feedback replies you may have left on them.", name: pseud.name) %>
```

Comment:
Why the change from "remove" to "detach"? (I'm thinking "remove" is easier for people who don't speak English as a first language.)

Reply:
I merged in a lot of the wording from the `orphans/_orphan_work` partial, which uses detach:
https://github.com/otwcode/otwarchive/blob/156720eb90c391d183f598edf531f940b0fd5fe3/app/views/orphans/_orphan_work.html.erb#L4-L8
Do you want me to change it to remove everywhere?

(Also, what about information vs. data? The work partial uses "data"; the user partial uses "information".)

Reply:
"Remove" everywhere would be great, thanks! And "data" should be good for the other one. (I ran the sentence through the [Readability Analyzer](https://datayze.com/readability-analyzer.php) our Docs team uses and it had a surprisingly higher level of reading ease with "data" than "information." I wasn't expecting that.)

Reply:
Remove and data it is!

(It may be related to the number of syllables, rather than the frequency of the word.)